bin/snyk_tests.sh

```diff
@@ -22,13 +24,39 @@ source bin/common/check_returncode.sh
 # authenticate snyk CLI locally
 run_and_check_returncode "snyk auth $SNYK_AUTHENTICATION_TOKEN"
 
-# Iac run limited to 20 runs a month. TBD Discuss with Jon Velapondi.
-# echo "Scanning Infrastructure as code with Snyk for potential vulnerabilities..."
-# snyk iac test || true
-
```

Comment:
based on discussion with Jon Velapondi on slack https://eosdis.slack.com/archives/CH9TDFEBS/p1673276829431219?thread_ts=1672768892.636039&cid=CH9TDFEBS,

"you get 30/month "for free".  For anything else, we'll need to pay extra for it. "
Decided to not use it for now due to limited availability.